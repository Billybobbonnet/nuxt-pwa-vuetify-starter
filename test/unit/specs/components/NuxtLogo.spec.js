import { mount } from "@vue/test-utils";
import NuxtLogo from "~/components/NuxtLogo";

describe("Nuxt logo", () => {
  it("renders to a snapshot", () => {
    const wrapper = mount(NuxtLogo);
    expect(wrapper.element).toMatchSnapshot();
  });
});
