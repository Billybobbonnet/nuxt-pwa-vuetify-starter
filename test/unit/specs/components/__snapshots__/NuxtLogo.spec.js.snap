// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nuxt logo renders to a snapshot 1`] = `
<div
  class="vue-to-nuxt-logo"
>
  <div
    class="triangle triangle-1"
  />
  <div
    class="triangle triangle-2"
  />
  <div
    class="triangle triangle-3"
  />
  <div
    class="triangle triangle-4"
  />
</div>
`;
